### DISCUSSION TITLE
配置成功后没有网络

### DISCUSSION THREAD

**Discussion by ChinryZhu** [Qv2ray/qv2ray.github.io#175] [Q&A]
确认使用的VPS服务器没有被墙，确认对应端口已开启

日志片段：
配置成功
[Vmess] 233v2.com_202.182.102.83(202***83:39213)
启动服务(2021/8/17 2:05:44)...
V2Ray 4.36.2 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.16 windows/386)
A unified platform for anti-censorship.
2021/08/17 02:05:44 [Info] github.com/v2fly/v2ray-core/v4/main/jsonem: Reading config: D:\test\v2rayN-Core\v2rayN-Core\config.json
2021/08/17 02:05:45 [Warning] github.com/v2fly/v2ray-core/v4: V2Ray 4.36.2 started
2021/08/17 02:05:56 127.0.0.1:57490 accepted //docs.qq.com:443 [proxy]
2021/08/17 02:06:06 127.0.0.1:55143 accepted //docs.qq.com:443 [proxy]
2021/08/17 02:06:08 127.0.0.1:53009 accepted //api.github.com:443 [proxy]
2021/08/17 02:06:08 127.0.0.1:63944 accepted //github.com:443 [proxy]
2021/08/17 02:06:17 127.0.0.1:51627 accepted //mtalk.google.com:443 [proxy]
2021/08/17 02:06:17 127.0.0.1:49623 accepted //github.com:443 [proxy]
2021/08/17 02:06:19 127.0.0.1:58876 accepted //github.githubassets.com:443 [proxy]
2021/08/17 02:06:19 127.0.0.1:60280 accepted //avatars.githubusercontent.com:443 [proxy]
2021/08/17 02:06:19 127.0.0.1:62468 accepted //collector.githubapp.com:443 [block]
2021/08/17 02:06:19 127.0.0.1:56800 accepted //github.com:443 [proxy]
2021/08/17 02:06:19 127.0.0.1:62942 accepted //collector.githubapp.com:443 [block]
2021/08/17 02:06:19 127.0.0.1:51726 accepted //api.github.com:443 [proxy]
2021/08/17 02:06:22 127.0.0.1:51595 accepted //api.vc.bilibili.com:443 [proxy]
2021/08/17 02:06:22 127.0.0.1:61914 accepted //s.url.cn:443 [proxy]
2021/08/17 02:06:22 127.0.0.1:61740 accepted //docs.qq.com:443 [proxy]
2021/08/17 02:06:26 127.0.0.1:58654 accepted //github.com:443 [proxy]
2021/08/17 02:06:29 127.0.0.1:56831 accepted //play.google.com:443 [proxy]
2021/08/17 02:06:29 [Warning] [817926076] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:39 [Warning] [1888249882] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:41 [Warning] [3931898279] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:41 [Warning] [660140066] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:42 127.0.0.1:62301 accepted //api.vc.bilibili.com:443 [proxy]
2021/08/17 02:06:50 [Warning] [1834703613] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:50 [Warning] [1707605281] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:52 [Warning] [4006126729] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:52 [Warning] [3621704656] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:52 [Warning] [3724313568] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:52 [Warning] [1417418109] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:54 127.0.0.1:57508 accepted //github.com:443 [proxy]
2021/08/17 02:06:54 127.0.0.1:61792 accepted //aegis.qq.com:443 [proxy]
2021/08/17 02:06:55 [Warning] [1507704269] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:55 [Warning] [672588327] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:55 [Warning] [816224983] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vmess/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [dial tcp 202.182.102.83:39213: i/o timeout dial tcp 202.182.102.83:39213: operation was canceled] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/08/17 02:06:56 127.0.0.1:50151 accepted //docs.qq.com:443 [proxy]

**Comment by cubercsl** [comment]
请不要滥用此讨论区讨论节点或其他软件问题。
